modules/ROOT/pages/migration-prep.adoc

```diff
@@ -10,28 +10,24 @@ Mule 4 introduces many changes, so plan accordingly before attempting to migrate
 [[when_to_start]]
 == When to Start Using Mule 4
 
-We recommend that you develop all *new* projects on Mule 4, provided that:
+MuleSoft recommends that you develop all *new* projects on Mule 4, provided that:
 
-* You and your team have updated your skills for Mule 4 via the documentation or formal training.
-* You have the proper deployment environment (see below).
-* You do not require all applications to be on a single version of Mule (see below).
+* You and your team have updated your skills for Mule 4 through the documentation or formal training.
+* You have the <<prepare_dev_environ, proper deployment environment>>.
 
-////
-DOCS-7411 removing for now per Eduardo Chicconi, among others:
-For projects which are already in development or deployed on Mule 3 (whether a prospect or customer), we recommend you wait to migrate these applications until the Mule 3 to Mule 4 migration tool is available. However, you can follow this guide to migrate manually.
-////
+For projects deployed on Mule 3, use the Mule Migration Assistant (MMA) to avoid re-creating the application in Mule 4.
 
 Migrate your applications if any one of the following conditions is met:
 
-* The 3.x version you're using reaches end of life
-* You want to make significant updates to the existing applications
-* You want to take advantage of key Mule 4 capabilities
-* You decide to upgrade all their apps to Mule 4 because you require all apps to be on one version (some onprem customers)
+* The 3.x version you are using reaches end of life.
+* You want to make significant updates to the existing applications.
+* You want to take advantage of key Mule 4 capabilities.
+* You decide to upgrade all your apps to Mule 4 because your organization requires all apps to be on one version (for some on-prem scenarios).
 
 [[prepare_dev_environ]]
 == Setting up Your Local Development Environment
 
-First, you will need to download Studio 7 and install it. If you're deploying locally, you will also need to download the Mule 4 standalone runtime.
+First, you need to download Studio 7 and install it. If you're deploying locally, you also need to download the Mule 4 standalone runtime, if the runtime that comes with Studio does not match the one you want to use for runtime purposes.
```

Comment:
> Download and install Anypoint Studio 7. If the Mule version that comes with Anypoint Studio does not match the Mule version you want to use for local deployments, download the desired 4.x version of Mule runtime engine.
(Please check if this rewording is better, it might need some tweaks also)

Reply:
done via suggestion

Reply:
done via suggestion